src/commands/misc/lorem.js

```diff
@@ -0,0 +1,30 @@
+const { SwitchbladeEmbed, Command, CommandError } = require('../../')
+const { LoremIpsum } = require('lorem-ipsum')
+
+module.exports = class Lorem extends Command {
+  constructor (client) {
+    super({
+      name: 'lorem',
+      parameters: [{
+        type: 'number',
+        full: true,
+        missingError: 'commands:lorem.noNumber',
+        required: false,
+        default: 2
```

Comment:
`default` is not an option.

```suggestion
        type: 'number',
        missingError: 'commands:lorem.noNumber',
        required: false,
        min: 1,
        max: 20,
```

PS. I don't know how many words each paragraph will have, so you may increase/decrease the max if you want

Reply:
What, default is not an option! But I think I saw it somewhere

Reply:
Probably my mistake. I thought that default is an option :(